dataverse-webapp/src/main/java/edu/harvard/iq/dataverse/harvest/client/HarvestingClientsPage.java

```diff
@@ -294,19 +283,11 @@ public void editClient(HarvestingClient harvestingClient) {
     public void deleteClient() {
         if (selectedClient != null) {
 
-            //configuredHarvestingClients.remove(selectedClient);
-
             logger.info("proceeding to delete harvesting client " + selectedClient.getName());
             try {
                 harvestingClientService.setDeleteInProgress(selectedClient.getId());
```

Comment:
both of those variables are very similar, if this is repository layer it should be *DAO.